Backend/app/rider-platform/rider-app/Main/src/Storage/Queries/PersonExtra.hs

```diff
@@ -99,6 +102,8 @@ updatePersonalInfo (Id personId) mbFirstName mbMiddleName mbLastName mbReferralC
         <> [Se.Set BeamP.clientConfigVersion (versionToText <$> mbClientConfigVersion) | isJust mbClientConfigVersion]
         <> [Se.Set BeamP.clientOsType ((.deviceType) <$> mbDevice) | isJust mbDevice]
         <> [Se.Set BeamP.clientOsVersion ((.deviceVersion) <$> mbDevice) | isJust mbDevice]
+        <> [Se.Set BeamP.clientModelName ((.deviceModel) <$> mbDevice) | isNothing person.clientDevice && isJust mbDevice]
```

Comment:
`isNothing person.clientDevice`

Why this check required ?

Reply:
we don't want to update these values everytime in case of update profile call. 
it was suggested by hemant